recipes/cupy/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "cupy" %}
+{% set version = "6.4.0" %}
+{% set sha256 = "ca70ca0823323f8c91d52eb304009eaa72feb44118864c1f00d8fa288352dc88" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not (linux64 and cuda_compiler_version == "9.2")]
+  ignore_run_exports:
+    - fastrlock
```

Comment:
Can you explain why?

Reply:
conda-build didn't seem to like this. Here's a snippet of the complaint it gave me. Here's the [full log]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=83348&view=logs&jobId=10bda42a-d420-5c28-a43f-bc4e66405e5c ).

```
   INFO (cupy): plugin library package conda-forge::fastrlock-0.4-py36hf484d3e_1000 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
```

Reply:
`fastrlock` doesn't even have a `run_exports` entry. Can you open an issue in conda-build?

Reply:
Sure. Raised issue ( https://github.com/conda/conda-build/issues/3784 ). Also cc'd you on the issue.

Though it looks like we can just drop this as the warning does not fail the build.